src/Libraries/Nop.Services/Orders/OrderNotifier.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Nop.Core.Domain.Logging;
+using Nop.Core.Domain.Orders;
+using Nop.Core.Domain.WebhookSettings;
+using Nop.Core.Models;
+using Nop.Services.Catalog;
+using Nop.Services.Customers;
+using Nop.Services.Events;
+using Nop.Services.Logging;
+using RestSharp.Portable;
+using RestSharp.Portable.HttpClient;
+
+namespace Nop.Services.Orders;
+
+public class OrderNotifier : IConsumer<OrderPlacedEvent>
+{
+    private readonly IOrderService _orderService;
+    private readonly ICustomerService _customerService;
+    private readonly IProductService _productService;
+    private readonly WebhookSettings _webhookSettings;
+    private readonly ILogger _logger;
+
+    public OrderNotifier(IOrderService orderService, ICustomerService customerService, IProductService productService,
+        ILogger logger, WebhookSettings webhookSettings)
+    {
+        _orderService = orderService;
+        _customerService = customerService;
+        _productService = productService;
+        _webhookSettings = webhookSettings;
+        _logger = logger;
+    }
+
+    public async Task HandleEventAsync(OrderPlacedEvent eventMessage)
+    {
+        var orderItems = await _orderService.GetOrderItemsAsync(eventMessage.Order.Id);
```

Comment:
try catch for this method, and in log add if something went wrong (aka Exception) with Message that will contain Order id and exception